app/main/views/suppliers.py

```diff
@@ -437,13 +437,27 @@ def do_search(search_query, offset, result_count, new_domains, framework_slug):
         else:
             sr_agg = postgres.array_agg(cast(func.substring(ServiceRole.name, 8), TEXT))
             q = q.having(sr_agg.contains(array(roles_list)))
-
+# recruiter
```

Comment:
I think we can remove these comments.  Your code is clear enough with the strings that are used.